blog/2025-09-24-gravitino-1-0-0-release-notes.mdx

```diff
@@ -0,0 +1,128 @@
+
+## Metadata-driven action system
+
+In 1.0.0, we have introduced 3 new components, with which we can build jobs to accomplish the metadata-driven actions, like table compaction, TTL data management, PII identification, etc. These 3 new components are: statistics system, policy system, and job system.
+
+Taking table compaction as an example:
+
+* Firstly, users can define the table compaction policy in Gravitino and associate this policy with the tables that need to be compacted.  
+* Then, users can save the statistics of the table to Gravitino.  
+* Also, users can define a job template for the compaction.  
+* Lastly, users can use the statistics with the defined policy to generate the compaction parameters and use these parameters to trigger a compaction job based on the defined job templates. 
+
+### Statistics system
+
+The statistics system is a new component for the statistics store and retrieval. You can define and store the table/partition level statistics in Gravitino, and also fetch them through Gravitino for different purposes.  
+For the details of how we design this component, please see [#7268](https://github.com/apache/gravitino/issues/7268). For how to use the statistics system, you can refer to the documentation [here](https://gravitino.apache.org/docs/1.0.0/manage-statistics-in-gravitino/).
+
+### Policy system
+
+The policy system helps you define action rules in Gravitino, like compaction rules or TTL rules. The defined policy can be associated with the entities, which means these rules will be enforced on the dedicated metadata. Users can leverage these enforced polices to decide how to trigger an action on the dedicated metadata.
+
+Please refer to the policy system [documentation](https://gravitino.apache.org/docs/1.0.0/manage-policies-in-gravitino) to know how to use it. If you want to know more about the implementation details of the policy system, please see [#7139](https://github.com/apache/gravitino/issues/7139).
+
+### Job system
+
+The job system is another system for you to submit and run jobs through Gravitino. Users can 
+register a job template, then trigger a job based on the specific job template. Gravitino will help to submit the job to the dedicated job executor, like Apache Airflow. Gravitino can manage the job lifecycle and save the job status in it. With the job system, users can run a self-defined job to accomplish a metadata-driven action system.
+
+The job system is still in active development. In 1.0.0, we have an initial version to support running the jobs as a local process. If you want to know more about the design details, you can follow issue [#7154](https://github.com/apache/gravitino/issues/7154). Also, a user-facing documentation can be found [here](https://gravitino.apache.org/docs/1.0.0/manage-jobs-in-gravitino).
```

Comment:
Suggested change: In version 1.0.0, we have an initial version that supports running jobs as a local process.